resources/views/store/edit_registration.blade.php

```diff
@@ -180,47 +180,43 @@ class="@if($errors->has('pri_carer'))invalid @endif" type="text"
                 </div>
                 @includeWhen(!empty($noticeReasons), 'store.partials.notice_box', ['noticeReasons' => $noticeReasons])
                 <button class="long-button submit" type="submit">Save Changes</button>
-                <div><hr class="col-break"></div>
-                <button class="long-button"
+                <!-- Button commented out in case it needs to be reinstated -->
+                <!-- <button class="long-button"
                         onclick="window.open( '{{ URL::route("store.registration.print", ["registration" => $registration]) }}'); return false">
                     Print a 4 week collection sheet for this family
-                </button>
+                </button> -->
                 <a href="{{ route("store.registration.voucher-manager", ['registration' => $registration ]) }}" class="link">
                     <div class="link-button link-button-large">
                         <i class="fa fa-ticket button-icon" aria-hidden="true"></i>Go to voucher manager
                     </div>
                 </a>
-            </div>
-        </form>
+                
+                @if (!isset($registration->family->leaving_on) )
 
-        @if (!isset($registration->family->leaving_on) )
-            <form action="{{ URL::route('store.registration.family',['registration' => $registration]) }}" method="post"
```

Comment:
This form hit a different endpoint?

Reply:
I think my formaction went missing with some markup shuffling

Reply:
Ah! I see you've merged the two forms and then got the second button to submit all the formdata to the update route too?

Reply:
is that why we do Ah! Is that why formNoValidate?

Reply:
hmm. this ticket says we support ie11 😨 which formNoValidate isn't a thing for 😬 

Should we have the two forms back?